uftrace.c

```diff
@@ -857,8 +857,13 @@ static int parse_option(struct uftrace_opts *opts, int key, char *arg)
 		}
 		break;
 
-	case OPT_disabled:
-		opts->disabled = true;
+	case OPT_trace:
+		if (!strcmp(arg, "on"))
+			opts->trace = TRACE_STATE_ON;
+		else if (!strcmp(arg, "off"))
+			opts->trace = TRACE_STATE_OFF;
+		else if (strcmp("arg", "on"))
```

Comment:
Wouldn't it be simply `else`?

Reply:
Yes I messed up when rebasing changes, sorry.